Octokit/Helpers/ApiUrls.cs

```diff
@@ -4457,5 +4457,15 @@ public static Uri Meta()
         {
             return "meta".FormatUri();
         }
+
+        public static Uri AllOrganizationCredentials(string org)
```

Comment:
Would you also mind adding method comments here?  Sorry I completely missed this during the initial review.